build.gradle.kts

```diff
@@ -0,0 +1,74 @@
+plugins {
+    id("com.exactpro.th2.gradle.base") version "0.0.6"
+    id("com.exactpro.th2.gradle.publish") version "0.0.6"
+
+    kotlin("jvm") version "1.8.22"
+    kotlin("kapt") version "1.8.22" apply false
+    id("org.jetbrains.kotlin.plugin.serialization") version "1.8.22" apply false
+
+    id("com.exactpro.th2.gradle.grpc") version "0.0.6" apply false
+    id("com.exactpro.th2.gradle.component") version "0.0.6" apply false
+}
+
+val cradleApiVersion by extra("5.3.0-dev")
+val javalin by extra("5.4.2")
+val commonVersion by extra("5.11.0-dev")
```

Comment:
suggestion: If you are trying to specify versions in one place and reuse them in other projects, probably the better way is to use the version catalog. Right now you have to write the version correctly in two places - here and in the app project. As a result you can easily make a mistake in variable names